.github/workflows/test-package-build.yml

```diff
@@ -52,16 +49,21 @@ jobs:
         run: |
           echo "MLFLOW_SKINNY=true" >> $GITHUB_ENV
 
-      - name: Build UI
-        if: ${{ matrix.type == 'full' }}
-        working-directory: mlflow/server/js
-        env:
-          # Prevent warnings (emitted from react-pdf) from being treated as errors
-          # https://github.com/wojtekmaj/react-pdf/issues/280
-          CI: false
+      - name: Create dummy index.html
         run: |
-          yarn install
-          yarn run build
+          mkdir -p mlflow/server/js/build
+          echo '
+          <!DOCTYPE html>
+          <html lang="en">
+            <head>
+              <meta charset="utf-8">
+              <title>MLflow</title>
+            </head>
+          <body>
+            MLflow
+          </body>
+          </html>
+          ' > mlflow/server/js/build/index.html
```

Comment:
What's the part code used for ?

Reply:
@WeichenXu123 It's dummy MLflow UI.

Reply:
<img width="475" alt="image" src="https://user-images.githubusercontent.com/17039389/163769552-64a41ae6-252d-4427-b4da-6364dd066404.png">

Reply:
This code basically means "prentend we ran yarn build"